packages/fxa-payments-server/src/lib/apiClient.test.ts

```diff
@@ -422,6 +422,29 @@ describe('API requests', () => {
       });
       requestMock.done();
     });
+
+    it(`POST {auth-server}${path} with coupon`, async () => {
+      const expected = { sourceCountry: 'US', subscription: { what: 'ever' } };
+      const requestMock = nock(AUTH_BASE_URL).post(path).reply(200, expected);
+
+      const paramsWithPromo = Object.assign(params, { promotionCode: 'TEST' });
+
+      expect(
+        await apiCreateSubscriptionWithPaymentMethod(paramsWithPromo)
+      ).toEqual(expected.subscription);
+
+      expect(
+        <jest.Mock>createSubscriptionWithPaymentMethod_PENDING
+      ).toBeCalledWith(metricsOptions);
```

Comment:
There should a 'promotionCode' event property here since it's was added at https://github.com/mozilla/fxa/pull/11469/files#diff-babc37caf979ff6df28097552ec48f037eafa5881b2e0940d9230c548363dc52R325

Reply:
updated